### PR TITLE
Improve layout

### PR DESCRIPTION
**Pull request by vweijsters**
Fixed layout issues in our code base, while testing SA1507 implementation.
(fixed all instances of SA1507, SA1516 and SA1518)

### CODE REVIEW

**Comment by macpak** [comment]
Changes look ok.

File: StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs [hunk 1]
```diff
@@ -21,6 +20,7 @@ public class SA1121UnitTests : CodeFixVerifier
             new Tuple<string, string>("object", nameof(Object)),
             new Tuple<string, string>("string", nameof(String))
         };
+
```
Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs]: :question: Is this supposed to be reported, or is it just a result of #548?

Comment (vweijsters) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs]: That is a result of SA1516 indeed (#548)

Comment (sharwell) [inline StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1121UnitTests.cs]: I received confirmation in #561 that these needed to be fixed.

**Pull request merged**